docs/geps/gep-01.rst

```diff
@@ -128,8 +144,10 @@ general naming considerations here.
 - There is a hierarchical structure to these parameters in that each of them is
   associated with a group (e.g., ``arbeitsl_geld``, ``kinderzuschlag``). These groups or
   abbreviations thereof do not re-appear in the name of the parameter.
-- Parameter names should be aligned with relevant column names. This makes searching for
-  identifiers easier and less error-prone.
+- Parameter names should be generally be aligned with relevant column names. However,
+  since the group is not repeated for the parameter, it is often better not to
+  abbreviate them (e.g., ``arbeitsl_geld_2["vermögensfreibetrag"]`` for the parameter
+  and ````arbeitsl_geld_2_vermög_freib_hh`` for a column derived from it).
```

Comment:
Not sure about that. Having the same term to search for seems to be relevant.

Reply:
Yeah, I thought I'd put the idea out there. The sentence *"No substantial changes."* was not good.

There are pros and cons either way. Readability of the yaml files is also important (will lose some of that as we restrict ourselves to 32 characters in many places :wink:) and it is not bad if there is some distinction between column names/parameters if they are not exactly the same. Example above as a case in point.

Let's postpone until the next (=hopefully last) big round of renaming. I won't suggest we merge this before that happens.

Reply:
Alright, let's postpone the discussion. Since each parameter should have both a `name` and `description` key, I would hope that readability would still be okay. But I need to check how much abbreviations there would be

Reply:
Yes, name might be enough, in case we display that e.g. in dashboard. Should review how well we are doing on that (consistency etc.) when going through the other stuff.

Reply:
Upon inspection, I think would leave it in. Check the [new example in 4983585](https://github.com/iza-institute-of-labor-economics/gettsim/pull/342/commits/498358554024c004caaeea7425da1504ea31f3d8), I would find it weird to write ``vermög_grundfreib`` instead of ``vermögensgrundfreibetrag`` just to make sure we use the same abbreviations. 

I'd also think that it is reasonably clear it is just a possibility and we may just as well use the abbreviations we use for the functions.

Reply:
Fine with me!